scriptler/findRunningPipelineJobs.groovy

```diff
@@ -0,0 +1,32 @@
+/*** BEGIN META {
+  "name" : "Find running pipeline jobs.",
+  "comment" : "Find all pipeline jobs running for at least a given amount of time (delay).",
+  "parameters" : [],
+  "core": "1.612.1",
+  "authors" : [
+    { name : "Arnaud Héritier" }
+  ]
+} END META**/
+
+import groovy.time.*
```

Comment:
I tend to personally prefer specific imports as opposed to wildcard imports.  It makes it more clear where a class is coming from.